oidc-ui/src/components/L1Biometrics.js

```diff
@@ -18,6 +18,7 @@ export default function L1Biometrics({
   param,
   authService,
   openIDConnectService,
+  handleMoreWaysToSignIn,
```

Comment:
can this method be renamed to handleBackButtonClick?

Reply:
resolve in this [f23c62b](https://github.com/mosip/esignet/pull/367/commits/f23c62bca823f172be40fc5ad7a41974c9ebf45b)